test/regression.uts

```diff
@@ -203,7 +203,7 @@ load_session()
 update_session()
 assert get_var("test_value") == "8.8.8.8" #test_value == "8.8.8.8"
 assert get_var("init_value") == 123
- 
+
```

Comment:
Please remove any non necessary / unrelated change. They make PRs hard to review and breaks the commit history / blame.